wiki/fr/fr-Central-Servers.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
This file doesn't seem to be translated. Could you please remove it?

Reply:
I'm working on the translation of all the pages/files. It could take a moment… ;)
What is the good way?

Reply:
Should be OK now.
I'll create a new individual pull request for each new translated files.

Reply:
You don't need to do it for each newly translated file but you can work in blocks.

Reply:
OK.